src/actions/arcActions.ts

```diff
@@ -163,24 +159,17 @@ export function redeemReputationFromToken(scheme: Scheme, addressToRedeem: strin
       };
       const gasEstimate = await arc.web3.eth.estimateGas(txToSign);
       txToSign.gas = gasEstimate;
-      console.log(`estimated gas cost           : ${gasEstimate  * gasPrice}`);
-      // console.log(txToSign);
       // if the gas cost is higher then the users balance, we lower it to fit
       const userBalance = await arc.web3.eth.getBalance(redeemerAddress);
-      console.log(`balance of ${redeemerAddress}: ${userBalance}`);
       if (userBalance < gasEstimate * gasPrice) {
         txToSign.gasPrice = Math.floor(userBalance/gasEstimate);
       }
-      console.log(`estimated gas cost after adjusting: ${gasEstimate  * txToSign.gasPrice}`);
-      const signedTransaction = await arc.web3.eth.accounts.signTransaction(txToSign, privateKey);
-      // console.log(signedTransaction);
+      // const signedTransaction = await arc.web3.eth.accounts.signTransaction(txToSign, privateKey);
```

Comment:
why is this commented out?

Reply:
because I removed the console.log which was the only place it was being used.  It is not my code and I didn't want to delete someone else's potentially useful/informative code.

Reply:
oh weird, I guess not being used

Reply:
wait, it is being used, right below, we send the signed transaction. why do you think its not being used?

Reply:
it _is_ used, see comment a few lines later

Reply:
Look again, it is not being used.

Reply:
My badliness, hastiness when I saw that the return value of `sendSignedTransaction` was not being used.  I have restored the line.